semgrep-core/src/runner/README.md

```diff
@@ -0,0 +1,90 @@
+# Notes
```

Comment:
I think this is not that long that it could go in `Run_semgrep.mli`.

Reply:
Also we had a document (maybe in Quip) with the "current" perf issues, we could have one in Notion too, and maybe it's a better place.

Reply:
I ended up putting it in the mli file; I like having it close to the code. Not sure I'd think to check Notion or keep it up to date